client/src/pages/sniffers/useEnterKeyPress.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { useEffect } from "react";
+
+const ENTER_KEY = "Enter";
+
+export const useEnterKeyPress = (
```

Comment:
What would happen if in a single page there are 2 event listeners for ENTER? probably they both react to it isn't it?

Reply:
I think that you are right, I replaced this hook with a function + now we are using the `onKeyDown` property on inputs and text fields so in this way, the function triggered by <kbd>enter</kbd> will be associated with the relevant element element

Reply:
Much better!